contributing/contributing-code.md

```diff
@@ -123,18 +123,12 @@ For further detail on integrating third-party services (e.g., fonts, embeds, etc
 
 ## Contributor License Agreement
 
-AMP requires all contributors to either sign an individual Contributor License Agreement or be covered by a corporate Contributor License Agreement in order to protect contributors and users in issues of intellectual property.
+AMP requires all contributors to agree to the OpenJSF Contributor License Agreement in order to protect contributors and users in issues of intellectual property.
 
-We recommend you handle signing/being covered by a CLA _before_ you send a PR to avoid problems, though this is not absolutely necessary until your code is ready to be merged in.
+To agree to the OpenJSF Contributor License Agreement, visit https://cla-assistant.io/ampproject/amphtml, read through the agreement, and click "Login with GitHub to Agree."
```

Comment:
@rsimha is it correct to link to ampproject/amphtml here?  I noticed after I clicked "Agree" there, if I substituted any other ampproject repo it said I had already signed.  Should we just point people to the overall ampproject org?

@tobie in case he knows as well

Reply:
The CLA text at https://cla-assistant.io/ampproject/amphtml isn't the right CLA text.

The right text is the individual CLA listed on https://openjsf.org/about/the-openjs-foundation-cla/ and more specifically this one: https://openjsf.org/wp-content/uploads/sites/84/2020/10/OpenJS-Foundation-ICLA-2020-09-23.pdf

/cc @rsimha

Reply:
Once this is fixed, though, you should be able to give just https://cla-assistant.io/ampproject/ as the URL.

Reply:
> Should we just point people to the overall ampproject org?

@mrjoro Yes, I think we should, because the CLA check was deployed for the organization and not separately for each repo.

> The CLA text at cla-assistant.io/ampproject/amphtml isn't the right CLA text.

@tobie, can you clarify further? The text that is currently in use was recommended by https://github.com/ampproject/meta/issues/60#issuecomment-681111001, which links to [this](https://openjsf.org/wp-content/uploads/sites/84/2019/11/OpenJS-Foundation-Individual-CLA-2019-11-15.pdf) document. Has the preferred OpenJS CLA text has changed recently? Either way, we certainly can change the text, but it will require everyone to re-sign the CLA.

Reply:
Yeah, getting the new CLA text approved was why this whole process took so long. I can't believe I mistakenly linked you to the previous one. It seems the new one hadn't yet been released at that point, and I didn't notice this when linking it.

Reply:
@mrjoro @tobie I just tried navigating to https://cla-assistant.io/ampproject (just the org name) and ended up at a 404 page. I think we need to use the full URL (https://cla-assistant.io/ampproject/amphtml). Can one of you check on this to confirm that I'm correct?

Reply:
@rsimha you need the trailing slash: https://cla-assistant.io/ampproject/ works, https://cla-assistant.io/ampproject doesn't.

Reply:
In that case, I think we'll need to explicitly hyperlink the text with the trailing slash. Otherwise, Github will drop the slash when it displays the URL (like it did in both your comments on this thread.)

Reply:
Oh, wow I'm not seeing that here. That's so weird.

Reply:
It could be something in my logged-in Chrome profile that's hiding the trailing slash. It's subtle behavior though!

Reply:
Wow, that's weird:

<img width="796" alt="Screen Shot 2020-10-23 at 21 35 17" src="https://user-images.githubusercontent.com/3392/97046511-c8421e80-1577-11eb-9d23-20cba84afeab.png">

Reply:
```suggestion
To agree to the OpenJSF Contributor License Agreement, visit https://cla-assistant.io/ampproject/amphtml/, read through the agreement, and click "Login with GitHub to Agree."
```